easy-ui-tokens/src/font/styles.json

```diff
@@ -118,6 +118,11 @@
         },
         "line_height": { "value": "{font.line_height.1.value}" }
       }
+    },
+    "responsive_mapping": {
```

Comment:
i don't love this name. basically headings 1-3 render as a size down on small screens. this is trying to capture that